toolchest_client/tools/api.py

```diff
@@ -109,8 +109,8 @@ def kraken2(output_path, inputs=[], database_name="standard", database_version="
 
     if read_one:
         inputs = [read_one]
-    if read_two:
-        inputs.append(read_two)
+        if read_two:
+            inputs.append(read_two)
```

Comment:
Not related to shi7, but formerly the R2 (only) could be added in paired-end mode.

Reply:
We might want to add a note to the docstring about this, but I think that the docs/docstrings are getting unwieldly and may in need of restructuring soon, so I'm fine with specifying this later.